metricbeat/module/pgbouncer/lists/_meta/data.json

```diff
@@ -0,0 +1,37 @@
+{
+    "@timestamp": "2024-08-04T16:40:51.299Z",
+    "event": {
+        "dataset": "pgbouncer.lists",
+        "duration": 3184900,
+        "module": "pgbouncer"
+    },
+    "metricset": {
+        "name": "lists",
+        "period": 10000
+    },
+    "pgbouncer": {
+        "lists": {
+            "users": 2,
+            "clients": {
+                "free": 47,
+                "used": 3,
+                "login": 0
+            },
+            "peers": 0,
+            "peer_pools": 0,
+            "dns_queries": 0,
```

Comment:
there are still some more fields like `dns`. can you look into this?
for all the modules.

Reply:
https://github.com/elastic/beats/pull/41174/commits/b34251289c128dee58bbe2c1e38d82794911bac7
https://github.com/elastic/beats/pull/41174/commits/15a98ab35419df4ed00227f8e2baa3cba779416b